package.json

```diff
@@ -131,7 +131,7 @@
     "qrcode": "1.5.4",
     "roboto-fontface": "0.10.0",
     "rrule": "2.8.1",
-    "sortablejs": "patch:sortablejs@npm%3A1.15.3#~/.yarn/patches/sortablejs-npm-1.15.3-3235a8f83b.patch",
+    "sortablejs": "patch:sortablejs@npm%3A1.15.6#~/.yarn/patches/sortablejs-npm-1.15.6-3235a8f83b.patch",
```

Comment:
This is referencing to a file called `sortablejs-npm-1.15.3-3235a8f83b.patch` when you change the version here too, the patch won't get applied anymore.